src/terra-dev-site/guide/Component Guides/RecreatingItemCollection.guide.mdx

```diff
@@ -0,0 +1,201 @@
+  }
+);
+```
+
+Each row needs a `key` which is a unique identifier, in our example it is 'row-0', 'row-1', etc. `isStriped` is set to 'true' every other row to give a close appearance to past item-collection.
+
+`discloseAction` is a special object for use when we select that row. `metaData` just contains a key for that row and `discloseLabel` is just a brief description the row such as 'Power Row 0 - Details'. `isDisclosed` is 'true' only when our component's state matches with the metadata key. `discloseCellIndex` is the index of the cell that provides the accessible link entry point.
+
+`onDisclose` is a simple state update function as seen here.
+
+```javascript
+const handleRowDisclose = (event, metaData) => {
+  event.preventDefault();
+  if (selectedKey !== metaData.key) {
+    setSelectedKey(metaData.key);
+  }
+};
+```
+
+For creating cells in each row we do this
+
+```javascript
+const createCellsForRow = cells => cells.map(cell => createCell(cell));
+const createCell = cell => ({ key: cell.key, children: cell.children, attrs: cell.attrs });
+```
+
+Cells are simple and just require a key, the children of said cell and then we use `attrs` to set `className` in our example like so.
+
+```javascript
+const attrs = { className: cx('content-display')};
+```
+
+## Creating a Cell Grid that looks like a Table
+
+Instead of using `<Table>`, you can use [CellGrid](/components/terra-cell-grid/cell-grid/cell-grid) to create another version of the default content when over a certain window size. This method requires a little bit of CSS to be put in place to get a closer match to `<Table>`'s default style. The basic content look of it looks like this.
+
+```javascript
+  const cellGridDisplay = (
+    <div className={cx('cellgrid-wrapper')}>
+      <CellGrid className={cx('row')}>
+        <Cell className={cx('start-accessory')} key="cell-0" width={{ static: { value: 40, unit: 'px' } }}>{startAccessory}</Cell>
+        <Cell className={cx('display')} key="cell-i" width={{ scalar: 1 }}>{display}</Cell>
+        ...
+      </CellGrid>
+      ...
+    </div>
+  );
+```
+
+In this example, the `<CellGrid>` effectively act as rows and doesn't require any props other than defining a class for it. The `<Cell>`'s are where the content is placed. In the example provided at the bottom of this page, the content has two accessories on either side of the main information. Setting these accessories `width` prop to `{ static: { value: 40, unit: 'px' } }` allows the accessories to only take up a small amount of space, while the rest of the `Cell`'s use `{ scalar: 1 }` for width to automatically take up an equal percentage of the remaining space.
+
+For the main styles it uses the following CSS.
+
+```css
+.cellgrid-wrapper {
+  border-bottom: 1px solid #c8cacb;
+}
+
+.row {
+  border-top: 1px solid #c8cacb;
+  display: flex;
+  min-height: 60px;
+}
+
+.row:nth-child(even) {
+  background: #f4f4f4;
+}
+```
+
+## Creating a read-only example using HTML table
```

Comment:
I'd start with this example, the simplest, and work up the the most complex.

Reply:
No longer applicable since other examples have been removed.